build.gradle

```diff
@@ -453,6 +453,8 @@ project('segmentstore:storage:impl') {
         testCompile project(':test:testcommon')
         testCompile project(path:':segmentstore:storage', configuration:'testRuntime')
         testCompile group: 'ch.qos.logback', name: 'logback-classic', version: qosLogbackVersion
+        testCompile group: 'io.dropwizard.metrics', name: 'metrics-core', version: dropWizardVersion
```

Comment:
Where does these two dependencies get used? I cannot see new imports in the code change.

Reply:
Initially, I found that required for the compilation to succeed. However, I have revisited this and I have seen that these are not necessary anymore. Removed them, thanks.